app/kubernetes-ingress-controller/2.0.x/guides/using-udpingress.md

```diff
@@ -0,0 +1,351 @@
+---
+title: UDPIngress with Kong
+---
+
+This guide walks you through deploying a simple [Service][svc] that
+listens for [UDP datagrams][udp], and exposes this service outside
+of the cluster using Kong Gateway.
+
+[svc]:https://kubernetes.io/docs/concepts/services-networking/service/
+[udp]:https://datatracker.ietf.org/doc/html/rfc768
+
+## Overview
+
+Some of the most common UDP-based services available on the internet are
+[DNS servers][dns]. These servers are an important part of the infrastructure
+of the internet, and are also [present by default][kubedns] in Kubernetes
+clusters to allow [Pods][pods] within the cluster to look up other pods' IP
+addresses by name.
+
+For this example, you will:
+* Deploy your own [CoreDNS][coredns] server, which is the default
```

Comment:
This shortlink doesn't resolve to anything in the output. Was this supposed to be [kubedns], or is there another link missing?

Reply:
I missed the link, good catch thank you now fixed in 1513aa547490817ee8ea05712cda81025c216b91